web/packages/core/src/ruffle-player.ts

```diff
@@ -644,6 +644,10 @@ export class RufflePlayer extends HTMLElement {
             !("url" in options) || typeof options.url === "string",
             "`url` must be a string"
         );
+        check(
```

Comment:
I do not understand the codebase well enough for this, but, will the user see a UI error if they mess up the settings?

Though, it might not be the responsibility of this PR to ensure that the user can see a UI error if that happens, but a new issue and PR.

I am not certain.

Is `check` meant to be used only for cases where there is a programmer error, or should it also be used for cases where UI-users have made a mistake? If the former, I am not certain that it is a good idea to use `check` here, unless the UI ensures that a UI-user cannot make a mistake and end up failing here.

Reply:
Will using the `in` operator here really work? Should it not instead be:

```!(options.preferredRenderer && options.forceRenderer),```